finance/proposal-calls/cycle3/fits-io-FR.md

```diff
@@ -0,0 +1,20 @@
+### Title
+`io.fits` developer
+
+### Project Team
+This is an open hire proposed by @eteq, but the `io.fits` maintainer (@saimn) would be a prime source of input on the search.
+
+### Project Description
+`io.fits` is one of the most important pieces of astronomy infrastructure.  However, it currently has only one maintainer (@saimn), which implies to a potential serious "bus factor"/single-point-of-failure. Existing members of the astropy community have generally not been receptive to working on this due to the complexity of the codebase (and the fact that for many the work is simply not much fun). This FR aims to support bringing a new developer into the community who might be more willing to support this sub-package, who may or may not also end up a long-term maintainer.
+
+Specific tasks would be drawn from [existing `io.fits` issues](https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Aio.fits). Note that a key provision for all work on this hirr, though, is to ensure that the contributions do not exceed the capacity of existing maintainers (esp @saimn), so both issue selection and rate would depend significantly on that. I.e., a hire would be expected to either select issues that are reviewable by a non-expert on `io.fits`, or at a slow enough rate to not exceed @saimn's capacity (or both).
+
+Note an open hire of this sort would be expected to continue in another year if all goes well. This proposal would this be a *start* at growing relevant expertise, and it might be multiple cycles before a fresh community member could become a co-maintainer
+
+
+### Approximate Budget
+Exact scope would be kept flexible (see emphasis above of not overwhelming maintainers). At a fiducial hourly rate of ~$120/hr and an assumption of ~2 issues per month over a year (assuming about half of the issues could be reviewed by a generalist and half by a specialist, keeping it to one review a month for @saimn at max), and assuming 4hr per issue, a starting estimate might be $11520
+
+#### Minimum budget
+
+In principle the true minimum is a *single* issue - ~$480k. But in practice the minimum.is whatever recoups the time spent on the seaexy as part of the open call.
```

Comment:
$480,000 per issue? Where do I sign up!! 😉

Reply:
Oh wait this is io.fits

Reply:
srsly though, is this a typo?

Reply:
I think he meant 480 for a single issue (120$/hr for 4 hrs per issue).